internal/namespaces/lb/v1/custom_backend.go

```diff
@@ -501,7 +503,44 @@ func interceptBackend() core.CommandInterceptor {
 
 		res, err := runner(ctx, argsI)
 		if err != nil {
-			return nil, err
+			type rawBodyResponse struct {
+				Details []struct {
+					Field   string `json:"field"`
+					Message string `json:"message"`
+					Reason  string `json:"reason"`
+				} `json:"details"`
+				Message string `json:"message"`
+				Type    string `json:"type"`
+			}
```

Comment:
Is there no existing structures like that from Scaleway Go SDK ? Seems like the standard structure for attribute errors.